extensions/search/schema.json

```diff
@@ -0,0 +1,45 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "schema.json#",
+  "title": "Search Extension",
+  "type": "object",
+  "description": "STAC Search Extension",
+  "additionalProperties": true,
+  "allOf": [
+    {
+      "$ref": "#/definitions/search_meta"
+    }
+  ],
+  "definitions": {
+    "search_meta": {
+      "properties": {
+        "properties": {
+          "type": "object",
+          "required": [
+            "next",
+            "returned"
+          ],
+          "properties": {
+            "next": {
+              "type": ["string", "null"]
+            },
+            "limit": {
+              "type": ["number", "null"]
```

Comment:
I guess number should be integer, or are we allowing floats? ;-)

Reply:
+1 should be `integer`

Reply:
yes, blame my ignorance with json schema, even though my IDE has autocomplete for these